marlowe-integration/src/Test/Integration/Marlowe/Local.hs

```diff
@@ -316,6 +315,7 @@ withLocalMarloweRuntime' MarloweRuntimeOptions{..} test = withRunInIO \runInIO -
     contractStore <- createContractStore ContractStoreOptions
       { contractStoreDirectory = resolveWorkspacePath workspace "contract-store"
       , contractStoreStagingDirectory = resolveWorkspacePath workspace "contract-staging-area"
+      , lockingSleepBetweenRetries = 100_000
```

Comment:
Presumably this is microseconds. I wonder if `lockingMicrosecondsBetweenRetries` might be more expressive and self documenting?